CONTRIBUTING.md

```diff
@@ -275,3 +275,52 @@ see <https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html>.
 
 With this, you should now have a working development setup! See
 [above](#building-and-testing-miri) for how to proceed working on Miri.
+
+## Advanced topic: Syncing with the rustc repo
+
+We use the [`josh` proxy](https://github.com/josh-project/josh) to transmit
+changes between the rustc and Miri repositories. For now, josh needs to be built
+from source. This downloads and runs josh:
+
+```rust
+git clone https://github.com/josh-project/josh
+cd josh
+git checkout @changes/master/christian.schilling.de@gmail.com/start-filter
```

Comment:
What is that filter 😃

Reply:
Oh that's a branch

Reply:
Yeah, it's the josh feature branch for the `at_commit` filter.

That filter is also a bit weird, but this is not it. ;)